src/main/scala/strawman/collection/Iterable.scala

```diff
@@ -218,6 +220,11 @@ trait ConstrainedIterablePolyTransforms[+A, +C[A], +CC[X] <: C[X], Ev[A]] extend
   def zip[B](xs: IterableOnce[B])(implicit ev: Ev[(A @uncheckedVariance, B)]): CC[(A @uncheckedVariance, B)] = constrainedFromIterable(View.Zip(coll, xs))
   // sound bcs of VarianceNote
 
+  def collect[B: Ev](pf: scala.PartialFunction[A, B]): CC[B] = flatMap(a => 
+    if (pf.isDefinedAt(a)) View.Elems(pf(a))
```

Comment:
Actually, I didn't even mean to leave this in.  I just wanted to check something about type inference.